lib/functions/function_tidy_measures.R

```diff
@@ -0,0 +1,28 @@
+#' Tidy measures data
+#'
+#' Creates a tidier dataframe of measures data.
+#' The measures must be named in a spedific way for this function to work properly.
+#'
+#' @param data A dataframe containing output from the OpenSAFELY measures framework
+#' @param pf_measures_name_mapping List, specifying the dict of measure names.
```

Comment:
param pf_measures_name_dict List, specifying the dict of measure names

Reply:
thanks :)

Reply:
addressed in a82ca2f46270574a6c65067f0e332b908e8a9cab